client/src/components/SidePanel/Builder/AssistantPanel.tsx

```diff
@@ -420,24 +421,32 @@ export default function AssistantPanel({
                   </button>
                 )}
                 {actionsEnabled === true && (
-                  <button
-                    type="button"
-                    disabled={!assistant_id}
-                    onClick={() => {
-                      if (!assistant_id) {
-                        return showToast({
-                          message: localize('com_assistants_actions_disabled'),
-                          status: 'warning',
-                        });
-                      }
-                      setActivePanel(Panel.actions);
-                    }}
-                    className="btn btn-neutral border-token-border-light relative h-8 w-full rounded-lg font-medium"
+                  <TooltipAnchor
+                    description={localize('com_assistants_actions_disabled')}
+                    className="relative w-full"
+                    showOnHover={!assistant_id}
+                    tabIndex={0}
                   >
```

Comment:
this creates an a11y issue. You can try to use the `asChild` (I don't remember if it works in the anchor) or set the render prop and put inside the button. Also, why using a toast? see ariakit's docs, you can use the `disabled` and `accessibleWhenDisabled` props instead of a toast